apps/src/music/blockly/toolbox.js

```diff
@@ -82,6 +82,7 @@ const toolboxBlocks = {
     },
   },
   [BlockTypes.PLAY_SOUNDS_TOGETHER]: {
+    id: 'play-sounds-together-block',
```

Comment:
Good idea.  Updated to use the `BlockTypes`.

The ID is assigned to `data-id`.  I did some experimentation and the behaviour is interesting...

The ID will be assigned to the block in the toolbox.  It will also be applied to the first instance of that block that's dragged into the workspace.  However, subsequent instances of the block that are dragged in will each be assigned unique IDs instead.

Similarly, if the starter code has the same ID specified multiple times, then the first block will get that ID, but subsequent blocks will instead be assigned unique IDs.